### DISCUSSION TITLE
Pnp - Search Filter. Location is displayed with delay or not at all

### DISCUSSION THREAD

**Discussion by BesimDuran** [microsoft-search/pnp-modern-search#3201] [Q&A]
Version used
Ex: 4.9.2

We have a problem with one variable (Location) when using the filter. All variables/user information we use work fine, except for the location. there we have a issue. Since we have about 2000 users we wanted to introduce a people search as a phone book. But here we have the problem that the location has problems to load the user information with the PnP- search filter. In the Pnp-Search results the information (also the location) is maintained.

the process of this we already created more variables (RefinableString). But there are about 10-15 new users per day - which would take about half a year with a number of 2000 users.

The variables are maintained and fully valid for each user. 

Do you have any idea why the information is only visible with such a delay?

many thanks

**Comment by kasperbolarsen** [comment]
Not sure that the issue is. Are you asking why new users takes a long time before they show up in the search results?

In that case it is most likely related to the search crawler isn't running often enough. The Crawler is MS property and the only way to handle that issue is to create a service request. Sometime they can do some kind of magic ( moving your tenant to a farm with less load?) that will resolve the issue.

**Reply by BesimDuran** [reply]
It's not just new users. But already existing users who are simply not included. 
Can you trigger the crawler manually?

If this doesnt work I'll open a ticket and ask directly MS. Thank you!

**Comment by kasperbolarsen** [comment]
No, the option from On-Premises where we could trigger a Full Crawl doesn't exists in Online. However, check out this script which is similar: https://pnp.github.io/script-samples/spo-request-pnp-reindex-user-profile/README.html?tabs=pnpps
